build.sh

```diff
@@ -299,11 +299,11 @@ if [[ "$WATTSI_RESULT" -ne "0" ]]; then
     echo "The correct line numbers for errors in the $HTML_SOURCE/source file are shown below:"
     echo
     cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
```

Comment:
Oh, sorry, I see that we output the wrong line numbers on line 295. But we should probably output both? So keep the "Found fatal errors. Stopping" inside the `if [[ "$WATTSI_RESULT" -ne "0" ]]`, and move lines 299-301 outside of it?